src/index.js

```diff
@@ -65,7 +64,7 @@ function setBuildNumber(usePackageJsonVersion, branch, gitHubToken, gitHubRepo,
 	}
 
 	console.log("##teamcity[blockOpened name='Pull Request']");
-	var pullRequestId = Number(pullRequestMatch[1]);
+	var pullRequestId = branch.includes("merge") ? (pullRequestMatch[1]) : (pullRequestMatch[2]);
```

Comment:
I think could simply be:
```suggestion
	var pullRequestId = pullRequestMatch[1] || pullRequestMatch[2];
```